src/test/java/uk/nhs/tis/trainee/notifications/service/HistoryServiceIntegrationTest.java

```diff
@@ -169,15 +169,15 @@ void shouldSortFoundNotificationsBySentAtWhenMultipleFound() {
 
     Instant now = Instant.now().truncatedTo(ChronoUnit.MILLIS);
     service.save(new History(null, tisReferenceInfo, FORM_UPDATED, recipientInfo, templateInfo,
-        now, now, SENT, null));
+        now, now, SENT, null, null));
 
     Instant before = SENT_AT.minus(Duration.ofDays(1));
     Instant after = SENT_AT.plus(Duration.ofDays(1));
     service.save(new History(null, tisReferenceInfo, FORM_UPDATED, recipientInfo, templateInfo,
-        before, after, SENT, null));
+        before, after, SENT, null, null));
```

Comment:
Having to chase these changes all over the app makes me regret using records at times 🤦🏻 

I think Lombok can add builders to records which might give us the best of both worlds (something for the backlog!).

Reply:
✔️ . It looks like IntelliJ didn't support that properly initially, but it was fixed a year ago or so.